src/bin/mosaic-start.ts

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
```

Comment:
I would propose not to have multiple chains as an input and stick with one.

Reply:
Can you please create a new ticket? In my opinion, this is definitely out of the scope of this ticket.